stream-bar/extension/StreamBarManager.ts

```diff
@@ -34,11 +36,27 @@ export class StreamBarManager extends Manager {
         });
         this.register(this.spotifyClient, "SpotifyClient", () => this.initSpotifyClient());
         this.register(this.streamelementsClient, "StreamelementsClient", () => this.initStreamelementsClient());
-
+		this.register(this.twitchClient, "TwitchClient", () => this.initTwitchClient());
         // TODO: Write frontend
-        // TODO: Add bot702 and !song
     }
 
+	async initTwitchClient(): Promise<void> {
+		await this.twitchClient?.getClient()?.join("#skate702");
+		this.twitchClient?.getClient()?.onMessage(async (channel, _, message) => {
+			if (message.startsWith("!song")) {
+				if (message.split(" ").length === 2) {
+					const target = message.split(" ")[1];
+					await this.retrieveCurrentSong();
+				this.twitchClient?.getClient()?.say(channel, `@${target} Current song is "${this.streamBarInfo.value.songName}" - "${this.streamBarInfo.value.artistName}"`);
```

Comment:
Just wondering: Are song names not usually displayed the other way around, meaning: `artist - song`?